jobs/deploy-fastly.groovy

```diff
@@ -83,6 +83,16 @@ def installDeps() {
    }
 }
 
+def _currentVersion() {
+   def cmd = [
+       "make",
+       "-C", "webapp/services/fastly-khanacademy",
+        params.TARGET == "prod" ? "current-version" : "current-version-test",
```

Comment:
Should be `active-version` right? https://github.com/Khan/webapp/pull/3223/files#diff-56d854eaca9ec15cf53462a7a76d28609a88cdc28e46716670deadbd7a0eb924R95

Reply:
Thanks!  Good catch.